### ISSUE TITLE
preventing infinite loops

### ISSUE THREAD

**Issue by user-74** [MajeedKazemi/blocks-to-code-old#7]
after code generation and before execution, we should inject some code that detects whether an infinite loop has been created, e.g. a max number of iterations/recursions, so we can break out of them and prevent the tab from crashing.

The repeat until / while blocks is the main point of concern here.

**Comment by MajeedKazemi** [comment]
Good catch, @user-74 . this is important to do as well. and injecting code (max number of iteration) is the way to go.
